app/store/models/Forum.ts

```diff
@@ -0,0 +1,50 @@
+import type { Comment } from './Comment';
+import type { PublicUser } from './User';
+import type { ContentTarget } from '../utils/contentTarget';
+import type { Dateish, ID } from 'app/models';
```

Comment:
```suggestion
import type { Dateish } from 'app/models';
import type { ID } from 'app/store/models'
```

Reply:
Ah this is why I needed to parseint, makes sense.